packages/@expo/config-plugins/src/ios/BuildProperties.ts

```diff
@@ -70,14 +70,30 @@ export const withNewArchEnabledPodfileProps = createBuildPodfilePropsConfigPlugi
   'withNewArchEnabledPodfileProps'
 );
 
+/**
+ * A config-plugin to update `ios/Podfile.properties.json` from the `updates.useNativeDebug` in expo config
+ */
+export const withUpdatesNativeDebugPodfileProps = createBuildPodfilePropsConfigPlugin<ExpoConfig>(
+  [
+    {
+      propName: 'updatesNativeDebug',
+      propValueGetter: (config) => (config?.updates?.useNativeDebug ?? false).toString(),
+      removePropWhenValueIsNull: true,
+    },
+  ],
+  'withUpdatesNativeDebugPodfileProps'
+);
+
```

Comment:
leaning toward to put updates related plugins together at https://github.com/expo/expo/blob/main/packages/%40expo/config-plugins/src/ios/Updates.ts.

Reply:
thanks! the `Updates.ts` is still in the config-plugins package. you can import `createBuildPodfilePropsConfigPlugin` from BuildProperties.ts into Updates.ts.

fwiw, expo-build-properties also uses the exported `createBuildPodfilePropsConfigPlugin` to build most things.

Reply:
Done